walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.replaceAll("\\s+","");
```

Comment:
What about an input like "      John Smith      " -> this will result in "Hello JohnSmith", you should trim the edges alone I suppose.